static/_redirects

```diff
@@ -1,8 +1,14 @@
 /nextflow           https://www.nextflow.io/docs/latest/ 301!
 
-# Fix top nav 404 for Cloud and Enterprise after index page rename
-/platform-cloud   /platform-cloud/platform-cloud 301
-/platform-enterprise/:version   /platform-enterprise/:version/platform-enterprise 301
+# These redirects reverse the existing 301's 
+# See https://moz.com/blog/can-you-reverse-a-301-redirect
+/platform-cloud/platform-cloud /platform-cloud 301
+/platform-enterprise/:version/platform-enterprise /platform-enterprise/:version 301
+/platform-enterprise/platform-enterprise /platform-enterprise 301
+
+# Drop the latest functionality redirect users to the "current version" which is the base URL.
+/platform-cloud/latest/platform-cloud /platform-cloud 301
+/platform-enterprise/latest/platform-enterprise /platform-enterprise 301
```

Comment:
@llewellyn-sl @justinegeffen @christopher-hakkaart these are the important items which needs review